lib/periodic_jobs.rb

```diff
@@ -150,7 +150,9 @@
   # Daily/weekly report of unsuccessful benefits intake submissions
   mgr.register('0 2 1 * *', 'VBADocuments::ReportMonthlySubmissions')
   # Monthly report of benefits intake submissions
-  mgr.register('0 2,9,16 * * 1-5', 'VBADocuments::SlackNotifier')
+  mgr.register('0 2,8,9,11,13,15,17,19,21 * * 1-5', 'VBADocuments::SlackNotifier')
```

Comment:
`0 */2 * * * ` Will run every 2 hours

Reply:
Since we already had concerns that `VBADocuments::SlackNotifier` was generating too much noise in the Slack channel, I don't think this schedule change will help that. Until we can replace the job, could we have it run 3 times a day still, but at something like 8am ET, 12pm ET, and 5pm ET, which are the hours where we'll have someone online?

Reply:
`0 8-17/3 * * *`

Reply:
The statuses we are reporting are in our database, not in datadog.
Even though we've tried a few different times, we could not get our slack production slack channel added to datadog.